extension.ts

```diff
@@ -30,7 +30,7 @@ let extensionContext: vscode.ExtensionContext;
 let previousActiveEditorId: EditorIdentity = new EditorIdentity();
 
 export async function getAndUpdateModeHandler(): Promise<ModeHandler> {
-  const [prevHandler] = await ModeHandlerMap.getOrCreate(previousActiveEditorId.toString());
```

Comment:
Maybe we can change this to a `create` function, but I can do that refactor in a later commit.